S3/Utils.py

```diff
@@ -33,8 +35,8 @@
     sys.stderr.flush()
     sys.exit(EX_OSFILE)
 
-import Config
-import Exceptions
+import S3.Config
```

Comment:
These are needed because of circular import dependency. If not for this reason `from . import Config` is certainly better.